cmd/demo/node.go

```diff
@@ -443,14 +461,23 @@ func (n *node) Info(args []string) error {
 	n.log.Traceln("Info...")
 
 	w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', tabwriter.Debug)
-	fmt.Fprintf(w, "Peer\tPhase\tVersion\tMy Ξ\tPeer Ξ\t\n")
+	fmt.Fprintf(w, "Peer\tPhase\tVersion\tMy Ξ\tPeer Ξ\tMy On-Chain Ξ\tPeer On-Chain Ξ\t\n")
 	for alias, peer := range n.peers {
+		ctx, cancel := context.WithTimeout(context.Background(), config.Chain.DeployTimeout)
+		defer cancel()
+		onChainBalListWei, err := getOnChainBal(ctx, n.onChain.Address(), peer.perunID)
```

Comment:
```suggestion
		onChainBals, err := getOnChainBal(ctx, n.onChain.Address(), peer.perunID)
```
Less verbose variable names. In general, don't use words like `...List`.